frontend_app_file/src/container/File.jsx

```diff
@@ -691,7 +691,7 @@ class File extends React.Component {
                   style={{
                     marginLeft: '5px'
                   }}
-                  faIcon={'fa-edit mr-3'}
+                  faIcon={'edit mr-3'}
```

Comment:
I missed this issue but never put another class into faIcon. It works, but you should not do that as mr-3 has nothing to do with the font-awesome icon